logical-comaprison-operators.js

```diff
@@ -47,33 +46,38 @@ const isHappy = false;
 
 // - Check if num is between 10 and 20 (inclusive) using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+ console.log(num >=10 && num <=20);
 // - Check if num is either less than 5 or greater than 50 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
-
+console.log(num >=5 || num <=50 );
 // - Check if str is either "apple" or "orange" using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(str == "apple" || str == "orange");
 
 // - Check if isHappy value is true using the logical NOT operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(isHappy == !isHappy);
 
 // - Check if num is even and greater than 10 using the logical AND operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(num / 2 == 0 && num > 10);
 
 // - Check if num is divisible by both 3 and 5 using the logical OR operator. Log the result to the console.
 // TODO: ADD YOUR CODE BELOW
+console.log(num / 3 ==0 || num / 5 == 0  )
```

Comment:
The division operator `/` will only return the result, not the remainder, here we're supposed to check if the number is divisible using the mod operator `%` and then comparing the remainder.